_config/config.yml

```diff
@@ -46,3 +46,4 @@ SilverStripe\Core\Injector\Injector:
     constructor:
       configuration: '%$SilverStripe\SearchService\Service\IndexConfiguration'
       registry: '%$SilverStripe\SearchService\Service\DocumentFetchCreatorRegistry'
+      service: '%$SilverStripe\SearchService\Interfaces\IndexingInterface'
```

Comment:
This appears to cause an infinite loop when doing dev/build?flush. I think it's related to: https://github.com/silverstripe/silverstripe-search-service/blob/869006d62fd7423ddc4bd25f4add1f5ea7054d7d/_config/appsearch.yml#L14
Any ideas how to resolve this mate?